schema/samcli.json

```diff
@@ -143,9 +143,175 @@
               "type": "object",
-              "properties": {}
+              "properties": {
+                "config_env": {
+                  "title": "config_env",
+                  "type": "string",
+                  "description": "Environment name specifying default parameter values in the configuration file.",
+                  "default": "default"
+                },
+                "config_file": {
+                  "title": "config_file",
+                  "type": "string",
+                  "description": "Configuration file containing default parameter values.",
+                  "default": "samconfig.toml"
+                },
```

Comment:
can the customer set these parameters in the config file, and how it will work if they can set them ??

Reply:
After some testing, it seems like those two parameters will get ignored anyways in the config file, though the debug menu still indicates that it's being run with the provided values. I'll include a check to disallow those parameters, since they'll otherwise cause more confusion about what's being read from where